client/rhel/spacewalk-client-tools/src/up2date_client/rhnregGui.py

```diff
@@ -960,7 +960,7 @@ def _provideCertificatePageApply(self):
                     errorWindow(rhnreg_constants.SSL_CERT_ERROR_MSG % (certFile, server_url))
 
                 return ERROR_WAS_HANDLED
-            except OpenSSL.SSL.Error:
+            except (OpenSSL.SSL.Error, SSL.SSL.SSLError):
```

Comment:
Why do you keep `OpenSSL.SSL.Error` here ?

Shouldn't this simply be:

```suggestion
            except SSL.SSL.SSLError:
```

Reply:
This code use both, SSL and OpenSSL. I have no idea what can be raised here.
Can we be sure that pyOpenSSL will not raise its own `Error` ?

Reply:
Ah, I see what you meant - we're calling some methods from the `pyOpenSSL.crypto` that might raise that custom pyOpenSSL exceptions.

Ok, looks good then :+1: